### ISSUE TITLE
Spinning Wild Mouse brakes inconsistency

### ISSUE THREAD

**Issue by Sdf2** [OpenRCT2/OpenRCT2#7709]
**OS: Windows 10**
**Version: v0.2.0-101**
**Commit/Build:0oec3ccd**

The brakes on Spinning Wild Mouse are inconsistent. The break values seem to be randomly changing between 2 values.
It's been like this in the past year and a half i've been playing on OpenRCT2.

**Steps to reproduce:**
1. Build a Spinning Wild Mouse ride with 3x pieces upward into a downhill 3x pieces into a break.
2. Complete the track.

**Screenshots / Video:**
![spinningwildmousebreak0](https://user-images.githubusercontent.com/40372939/41566557-5f04831e-735c-11e8-9f4b-c5fff6553db2.png)
![spinningwildmousebreak1](https://user-images.githubusercontent.com/40372939/41566559-603234a2-735c-11e8-9aa0-e6c3d5953cc7.png)

**Comment by Gymnasiast** [comment]
You know, it helps if you tell the value those brakes are set to in the first place.

**Comment by duncanspumpkin** [comment]
Out of interest is it every other vehicle that brakes differently.

**Comment by deurklink** [comment]
This does not only happen for spinning wild mouse, I also noticed two dueling coasters often going out of sync after a brake track because they would end up with slightly different speeds. I don't know if this is a bug or that it was intended.

**Comment by Sdf2** [comment]
As "deurlink" wrote, standard brakes seem to be like random at times. I kept trying out different cars on the same track. For example going into a 28 km/h break at 37 km/h speed the a Hyper Twister car would slow down to 27 km/h but sometimes to 24 km/h, a Giga Coaster car seems to be always braking to 27 km/h however on a Spinning Wild Mouse car at 35 km/h into a 28 km/h break nets you a 27 km/h speed or sometimes a 19 km/h speed and then the ride ends up crashing...

(I can't run the original game atm cos' i don't have DirectPlay or any other legacy stuff on this specific machine.
In order to run the game, i copied an old install to one of my drives and pointed the OpenRCT2Launcher to that directory.
I can try to install the game through Steam but i doubt that it would work without the legacy stuff ~ DirectPlay.)

**Comment by jensj12** [comment]
Happens in the original game as well. I had a brake set to 6km/h on a vertical drop coaster (1car/train), and some vehicles almost came to a complete stop. The speed most likely overshoots the target value (both brakes and boosters), which is noticeable with these light cars.

**Comment by Patrik356b** [comment]
Does the result even out with multiple brakes?

**Comment by LordMarcel** [comment]
![image](https://user-images.githubusercontent.com/25930360/41738938-dad18300-7593-11e8-8192-af60d94bfc08.png)
Multiple brakes do not help. The trains on this coaster will usually slow down to 3 km/h, but every now and then they slow down to 6 km/h. The brakes are set to 6 km/h.

**Comment by Sdf2** [comment]
Yes cos' if the first brake slows the car down to 3 km/h then the rest of the brakes do nothing?

**Comment by Gymnasiast** [comment]
No, of course not. They're set to slow down the train to 6 km/h and the train is already below that.

**Comment by Patrik356b** [comment]
I was more curious about how this affects a faster train, because the brakes can only slow down the train by X on a particular tile.
